### PR TITLE
feat: RSS-ECOMM-1_06_07 Create commercetools project and API for commercetools project.

### PR DESCRIPTION
**Pull request by Pavel-Kuvshinov**
## Overview
Create commercetools project and API for commercetools project.

**What has been done:**
- Create commercetools project with basic data in the Merchant Center;
- Create API for commercetools project;
- Add .env file with basic API data;

**Why this has been done:**
- Without this changes this project would not work.

**Issue(s) closed:**
- Closes RSS-ECOMM-1_06-RSS-ECOMM-1_07 - Create the project in CommerceTools and Create the API client for CommerceTools

## Type of Change
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### CODE REVIEW

File: kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+CTP_PROJECT_KEY=kick-flip-v2-webstore-warrioirs
```
Comment (makarovaiuliia) [inline kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env]: Есть возможность поправить ссылку и заменить "warrioirs" на "warriors"? Но без того, чтоб начинать всю настройку с самого начала

File: kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+CTP_CLIENT_SECRET=3cc2r3ii_v_yBZUucbgWSNfg9gqQ3W4v
+CTP_CLIENT_ID=T_WuEU3UsZl7PDMxDQFPlXGl
+CTP_AUTH_URL=https://auth.europe-west1.gcp.commercetools.com
+CTP_API_URL=https://api.europe-west1.gcp.commercetools.com
+CTP_SCOPES=manage_project:kick-flip-v2-webstore-warrioirs
```
Comment (makarovaiuliia) [inline kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env]: И тут

File: kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env (hunk 1 above)
Comment (Pavel-Kuvshinov) [reply kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env]: I fixed it.

File: kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env (hunk 2 above)
Comment (Pavel-Kuvshinov) [reply kickFlip-project/kick-flip-v2-webstore-warrioirs_kick-flip-API.env]: I fixed it.

**Review by makarovaiuliia — APPROVED** [review]

**Pull request merged**